bin/pygrb/pycbc_pygrb_exclusion_dist_table

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+
+# Copyright (C) 2024 Jacob Buchanan
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+"""Create table of exclusion distances."""
+
+import sys
+import argparse
+import json
+import pycbc.version
+import pycbc.results
+
+
+__author__ = "Jacob Buchanan <jacob.buchanan@ligo.org>"
+__version__ = pycbc.version.git_verbose_msg
+__date__ = pycbc.version.date
+__program__ = "pycbc_pygrb_exclusion_dist_table"
+
+parser = argparse.ArgumentParser(description=__doc__, formatter_class=
+                                 argparse.ArgumentDefaultsHelpFormatter)
+parser.add_argument("--version", action="version", version=__version__)
+parser.add_argument("--input-files", nargs="+", required=True,
+                    help="List of JSON input files" +
+                    " output by pycbc_pygrb_efficiency" +
+                    " containing exclusion distances.")
+parser.add_argument("--output-file", required=True,
+                    help="HTML output file containing table" +
+                    " of exclusion distances.")
+opts = parser.parse_args()
+
+# Load JSON files as a list of dictionaries
+file_contents = []
+for file_name in opts.input_files:
+    with open(file_name, "r") as file:
+        file_contents.append(json.load(file))
+
+# Get list of trials (i.e. OFFTRIAL_i, ONSOURCE, etc.)
+trials = []
+for fc in file_contents:
+    trials.append(fc["trial_name"])
+# Check that at least one trial exists
+if len(trials) == 0:
+    raise ValueError("No trials found in input files.")
```

Comment:
Should we make `--trial-name` required in `pycbc_pygrb_efficiency` to prevent this?  This way we have a check in place here for the standalone script, and a way to dodge the error in the workflow.

Reply:
Now that I looked better, `pycbc_pygrb_efficiency` must check that `opts.trial_name` is not None when `exclusion_dist_output_file` is not None.